tests/helpers.py

```diff
@@ -70,12 +70,41 @@ class FakeRelation(object):
         relation = FakeRelation(rel)
         self.relation_get.side_affect = relation.get
         passwd = self.relation_get('password', rid='mysql:0', unit='mysql/0')
+
+    set_relation_context can be used to simulate being in relation hook
+    context, eg omitting a relation id or unit implies the query is against
```

Comment:
I think it would be more clear to say: set_relation_context can be used to simulate being in a relation hook context. This allows omitting a relation id or unit when calling relation helpers as the related unit is present